Marlin/configuration_store.cpp

```diff
@@ -2209,14 +2196,12 @@ void MarlinSettings::reset() {
     /**
      * Probe Offset
      */
-    #if HAS_BED_PROBE
-      if (!forReplay) {
-        CONFIG_ECHO_START;
-        SERIAL_ECHOLNPGM("Z-Probe Offset (mm):");
-      }
+    if (!forReplay) {
       CONFIG_ECHO_START;
-      SERIAL_ECHOLNPAIR("  M851 Z", LINEAR_UNIT(zprobe_zoffset));
-    #endif
+      SERIAL_ECHOLNPGM("Z-Probe Offset (mm):");
+    }
+    CONFIG_ECHO_START;
+    SERIAL_ECHOLNPAIR("  M851 Z", LINEAR_UNIT(zprobe_zoffset));
```

Comment:
Should there be a "Z Probe Offset" option on a Cartesian with no probe?

Reply:
Maybe having the possibility to babystep a non probe related z_offset would be a plus; for the time being I only kept the homing change